bin/run_gis_csc.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+import asyncio
+from lsst.ts.gis import GISCsc
+
+asyncio.run(GISCsc.amain(index=False))
```

Comment:
Likewise, does this CSC need to be indexed? If not then please clean this up a bit.